tests/requirements/py3.txt

```diff
@@ -19,3 +19,5 @@ sqlparse >= 0.5.0
 tblib >= 3.0.0
 tzdata
 colorama >= 0.4.6; sys_platform == 'win32'
+diff-cover >= 8.0.0
+coverage[toml]
```

Comment:
These are specific to 1 GitHub action, so I would just install them directly in the GitHub action (rather than making everyone install as a testing requirement)